plugins/bcc-login/includes/class-bcc-login-visibility.php

```diff
@@ -1048,23 +1048,40 @@ function post_group_tags_widget($atts) {
         $post_id = get_the_ID();
         if (!$post_id) return false;
 
-        $attributes = shortcode_atts(array('limit' => 1000, 'with_link' => false), $atts);
+        $attributes = shortcode_atts(
+            array(
+                'limit' => 100,
+                'with_link' => true,
+                'personal' => false
+            ),
+            $atts
+        );
+
+        // Convert to actual booleans
+        $personal = filter_var($attributes['personal'], FILTER_VALIDATE_BOOLEAN);
```

Comment:
would something like "only_users_own_groups" or "only_own_groups" or "only_users_groups" be a little more explicit?

personal="true" is a little bit abstract in my opinion